### PR TITLE
Sdl rc baseline

### PR DESCRIPTION
**Pull request by OlesiaV**
HMI guidelines for GetInteriorVehicleData and OnInteriorVehicleData

### CODE REVIEW

File: docs/Common/Structs/index.md [hunk 1]
```diff
@@ -411,10 +411,67 @@
...
+|ventilationMode|VentilationMode|false|||
+
+### Temperature
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|unit|TemperatureUnit|true||Temperature Unit|
+|value|Float|true||The temperature value is in TemperatureUnit specified unit|
+
 ### ExternalConsentStatus
```
Comment (yang1070) [inline docs/Common/Structs/index.md]: how ExternalConsentStatus is used?

File: docs/RC/GetInteriorVehicleData/index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+### Example Request
+
+```json
+{
+    "id": 122,
+    "jsonrpc": "2.0",
+    "method": "RC.GetInteriorVehicleData",
+    "params": {
+        "appID": :1093636990,
```
Comment (yang1070) [inline docs/RC/GetInteriorVehicleData/index.md]: extra :

File: docs/RC/GetInteriorVehicleData/index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        "moduleType": "CLIMATE"
+        "subscribe": true
```
Comment (yang1070) [inline docs/RC/GetInteriorVehicleData/index.md]: why ":" is highlighted in red color?

File: docs/Common/Structs/index.md (hunk 1 above)
Comment (OlesiaV) [inline docs/Common/Structs/index.md]: It is for External User Consent Settings feature. Files Enums and Structs contain info for all SDL features not only for RC

**Comment by JackLivio** [comment]
Please add new files to config.json so the new pages can be navigated to on developer portal.

File: docs/RC/GetInteriorVehicleData/index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|moduleData|[Common.ModuleData](https://github.com/OlesiaV/https://github.com/OlesiaV/sdl_hmi_integration_guidelines/blob/master/docs/Common/Structs/index.md)|true||
+|isSubscribed|Boolean|false||
+
+### Sequence Diagrams
+|||
```
Comment (JackLivio) [inline docs/RC/GetInteriorVehicleData/index.md]: This chart is not displaying properly. I think there needs to be a line between lines 31 and 32

File: docs/RC/GetInteriorVehicleData/index.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+GetInteriorVehicleData
+![GetInteriorVehicleData](assets/GetInteriorVehicleData.png)
+
```
Comment (JackLivio) [inline docs/RC/GetInteriorVehicleData/index.md]: Also remove this line break

File: docs/RC/GetInteriorVehicleData/index.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+### Example Error
+
+```json
+{
+    "error": {
+        "code": 26,
+        "data": {
+            "method": "RC.GetInteriorVehicleData"
+        },
+        "message": "Read only parameters received"
```
Comment (JackLivio) [inline docs/RC/GetInteriorVehicleData/index.md]: When would this error message apply to "GetInteriorVehicleData"?

File: docs/RC/GetInteriorVehicleDataConsent/index.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+: SDL
+
+Purpose
+: Request for driver's permission to RC module for the specified application
+
+RC.GetInteriorVehicleDataConsent must be sent to HMI in case application in HMILevel FULL requested access to RC module that is already in use by another application. 
+HMI is expected to display a permission prompt to the driver showing the RC module and app details (for example, app's name).
+The driver is expected to have an ability to grant or deny the permission.
+
+!!! must
```
Comment (JackLivio) [inline docs/RC/GetInteriorVehicleDataConsent/index.md]: I don't think "!!! must" is the right tag for these bullet points. These sentences are not descriptive for what the HMI must do...

File: docs/RC/GetInteriorVehicleDataConsent/index.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+        "allowed": true,
+        "code": 0,
+        "method": "RC.GetInteriorVehicleDataConsent"
+    }
+}
+```
+
+### Example Error
+
+```json
```
Comment (JackLivio) [inline docs/RC/GetInteriorVehicleDataConsent/index.md]: This is not a correct error message

File: docs/RC/OnInteriorVehicleData/index.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+!!!
+
+### Notification
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|moduleData|[Common.ModuleData](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/master/docs/Common/Structs/index.md)|||
```
Comment (JackLivio) [inline docs/RC/OnInteriorVehicleData/index.md]: Module data is mandatory

File: docs/RC/OnInteriorVehicleData/index.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+
+### Sequence Diagrams
+||| OnInteriorVehicledata
```
Comment (JackLivio) [inline docs/RC/OnInteriorVehicleData/index.md]: Fix formatting for chart. Chart not displaying correctly.  See here https://smartdevicelink.com/en/docs/pull_request/f3f90b16d50ad6dbce7a00bbc59f9978495ebd0e/rc/oninteriorvehicledata/

File: docs/RC/OnRemoteControlSettings/index.md [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:---------|
+|allowed|Boolean|false||If "true" - RC is allowed; if "false" - RC is disallowed|
+|accessMode|[Common.RCAccessMode](https://github.com/OlesiaV/sdl_hmi_integration_guidelines/blob/2fecda9e5f0f4344a000092043c0530556a6bb8d/docs/Common/Enums/index.md)|false||The remote control access mode specified by the driver via HMI|
+
+### Sequence Diagrams
+
+|||
+OnRemoteControlSettings
```
Comment (JackLivio) [inline docs/RC/OnRemoteControlSettings/index.md]: Fix formatting. Charts are not displaying. https://smartdevicelink.com/en/docs/pull_request/f3f90b16d50ad6dbce7a00bbc59f9978495ebd0e/rc/onremotecontrolsettings/

File: docs/RC/ButtonPress/index.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+
+Sender
+: SDL
+
+Purpose
+: Emulate button press event on HMI for the common climate or radio control buttons in vehicle
+
+ButtonPress represents a request from an application to change settings of requested RC module by pressing the appropriate button.
+This RPC can be sent to the HMI from an application that is registered with REMOTE_CONTROL appHMIType and in one of the following states: FULL, LIMITTED, BACKGROUND.
+Module signed by the application in such reques has to be available on HMI and allowed for control change settings.
```
Comment (JackLivio) [inline docs/RC/ButtonPress/index.md]: `request` spelling typo

File: docs/RC/SetInteriorVehicleData/index.md [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+Type
+: Function
+
+Sender
+: SDL
+
+Purpose
+: Set RC module settings
+
+RC.SetInteriorvehicledata represents a request from an application to change settings of requested RC module. This RPC can be sent to the HMI for an application that is registered with REMOTE_CONTROL appHMIType and in one of the following states: FULL, LIMITTED, BACKGROUND. Module signed by the application in such reques has to be available on HMI and allowed for control change settings
```
Comment (JackLivio) [inline docs/RC/SetInteriorVehicleData/index.md]: `request` spelling typo

File: docs/RC/ButtonPress/index.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+### Response
+
+#### Parameters
+
+This RPC has no additional parameter requirements
+
+### Sequence Diagrams
+
+|||
+ButtonPress
```
Comment (JackLivio) [inline docs/RC/ButtonPress/index.md]: sequence diagram not displaying https://smartdevicelink.com/en/docs/pull_request/f3f90b16d50ad6dbce7a00bbc59f9978495ebd0e/rc/buttonpress/

File: docs/RC/OnRemoteControlSettings/index.md [hunk 15]
```diff
@@ -0,0 +1,47 @@
+## OnRemoteControlSettings
+
+Type
+: Notification
+
+Sender
+: HMI
+
+Purpose
+: Notification about remote-control settings and/or access mode changed. Sent after User`s choice through HMI.
```
Comment (JackLivio) [inline docs/RC/OnRemoteControlSettings/index.md]: used wrong character for apostrophe

File: docs/RC/GetInteriorVehicleDataConsent/index.md (hunk 7 above)
Comment (LuxoftAKutsan) [reply docs/RC/GetInteriorVehicleDataConsent/index.md]: @JackLivio do you mean steps should describe HMI behavior?

File: docs/RC/GetInteriorVehicleDataConsent/index.md (hunk 8 above)
Comment (LuxoftAKutsan) [reply docs/RC/GetInteriorVehicleDataConsent/index.md]: @JackLivio will replace with TIME_OUT

File: docs/RC/GetInteriorVehicleData/index.md (hunk 4 above)
Comment (LuxoftAKutsan) [reply docs/RC/GetInteriorVehicleData/index.md]: @JackLivio fixed

File: docs/RC/GetInteriorVehicleData/index.md (hunk 5 above)
Comment (LuxoftAKutsan) [reply docs/RC/GetInteriorVehicleData/index.md]: @JackLivio fixed

File: docs/RC/GetInteriorVehicleData/index.md (hunk 6 above)
Comment (LuxoftAKutsan) [reply docs/RC/GetInteriorVehicleData/index.md]: @JackLivio fixed

File: docs/RC/GetInteriorVehicleDataConsent/index.md (hunk 8 above)
Comment (LuxoftAKutsan) [reply docs/RC/GetInteriorVehicleDataConsent/index.md]: @JackLivio fixed

File: docs/RC/OnInteriorVehicleData/index.md (hunk 9 above)
Comment (LuxoftAKutsan) [reply docs/RC/OnInteriorVehicleData/index.md]: @JackLivio fixed

File: docs/RC/ButtonPress/index.md (hunk 12 above)
Comment (LuxoftAKutsan) [reply docs/RC/ButtonPress/index.md]: @JackLivio fixed

File: docs/RC/SetInteriorVehicleData/index.md (hunk 13 above)
Comment (LuxoftAKutsan) [reply docs/RC/SetInteriorVehicleData/index.md]: @JackLivio fixed

File: docs/RC/OnRemoteControlSettings/index.md (hunk 15 above)
Comment (LuxoftAKutsan) [reply docs/RC/OnRemoteControlSettings/index.md]: @JackLivio fixed

File: docs/RC/GetInteriorVehicleData/index.md [hunk 16]
```diff
@@ -0,0 +1,103 @@
...
+            "moduleType": "CLIMATE"
+        }
+    }
+}
+
+```
+
+### Example Error
+
+```jsonrpc
```
Comment (JackLivio) [inline docs/RC/GetInteriorVehicleData/index.md]: Remove rpc, should just be `json`

File: docs/RC/GetInteriorVehicleDataConsent/index.md (hunk 7 above)
Comment (JackLivio) [reply docs/RC/GetInteriorVehicleDataConsent/index.md]: Yes the must should describe the HMI's absolute responsibilities when handling this rpc. for example, HMI `must` prompt user to make selection for corresponding App and module after receiving GetInteriorVehicleDataConsent.

Comment (JackLivio) [reply docs/RC/GetInteriorVehicleDataConsent/index.md]: @LuxoftAKutsan ^

File: docs/RC/OnInteriorVehicleData/index.md (hunk 10 above)
Comment (JackLivio) [reply docs/RC/OnInteriorVehicleData/index.md]: fixed

File: docs/RC/ButtonPress/index.md [hunk 17]
```diff
@@ -0,0 +1,94 @@
...
+!!!
+
+### Request
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:---------|
+|moduleType|[Common.ModuleType](/docs/Common/Enums/index.md)|true||The module where the button should be pressed|
+|buttonName|[Common.ButtonName](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/SDL_RC_baseline_part2/docs/Common/Enums/index.md)|true|||
```
Comment (JackLivio) [inline docs/RC/ButtonPress/index.md]: please change to relative path. SDL_RC_baseline_part2 pr might not exist forever

File: docs/RC/ButtonPress/index.md [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+|buttonPressMode|[Common.ButtonPressMode](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/SDL_RC_baseline_part2/docs/Common/Enums/index.md)|true||Indicates whether this is a LONG or SHORT button press event.|
```
Comment (JackLivio) [inline docs/RC/ButtonPress/index.md]: please change to relative path. SDL_RC_baseline_part2 pr might not exist forever

File: docs/RC/SetInteriorVehicleData/index.md [hunk 19]
```diff
@@ -0,0 +1,37 @@
+## SetInteriorVehicleData
```
Comment (JackLivio) [inline docs/RC/SetInteriorVehicleData/index.md]: Corresponding charts to setinteriorvehicledata are not used?

File: docs/RC/GetInteriorVehicleDataConsent/index.md [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+Purpose
+: Request for driver's permission to RC module for the specified application
+
+RC.GetInteriorVehicleDataConsent must be sent to HMI in case application in HMILevel FULL requested access to RC module that is already in use by another application. 
+HMI is expected to display a permission prompt to the driver showing the RC module and app details (for example, app's name).
+The driver is expected to have an ability to grant or deny the permission.
+
+!!! must
+1. HMI must prompt user to make selection of resource allocation
+2. Respon to SDL with user choice within RC.GetInteriorVehicleDataConsent response
```
Comment (JackLivio) [inline docs/RC/GetInteriorVehicleDataConsent/index.md]: Respond*

**Pull request merged**